### DISCUSSION TITLE
Ng-Packagr way of building libraries - with support for multiple formats

### DISCUSSION THREAD

**Discussion by Lonli-Lokli** [nrwl/nx#23242] [Feature Requests]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md))

## Description
nx publish should generate several output formats (commonjs, esm) at once

## Motivation
Right now if I want to publish typescript library to npm I have to choose either commonjs or later format.

It would be great to have multiple versions published, for different consumers
## Suggested Implementation
generate multiple output formats, either with rollup or tsc

**Comment by Lonli-Lokli** [comment]
I still think it's a valuable feature

**Comment by jaysoo** [comment]
We're not planning on this feature, however if there is enough interest we will re-evaluate.
